src/scene/shader-lib/programs/lit-shader.js

```diff
@@ -1342,47 +1340,45 @@ class LitShader {
                 }
 
                 // specular / clear coat
-                if (light.affectSpecularity) {
```

Comment:
just double checking this was line removed on purpose, as @ellthompson recently added it if I remember correctly